checks/prometheus_rule.py

```diff
@@ -0,0 +1,47 @@
+from lib.base import CheckBase
+
+
+class CheckPrometheusRuleSeverity(CheckBase):
+    whitelist = ['PrometheusRule']
+
+    enable_parameter = 'prometheus-rule-severity'
+    description = 'check that PrometheusRule severity is in allowed values'
+    default_enabled = True
+
+    def check_severity(self, m):
+        allowed_severities = ['high', 'warning', 'medium', 'info']
```

Comment:
I think having two ways to deploy the same resource is gonna cause some added confusion. I appreciate that this is gradual, but I'd rather have a single way and give the tenants full control. Wdyt? 

PS: validating the severity values is still a good thing, since we can enforce our naming standards for them

Reply:
i don't see any harm in deploying the same resources from multiple places. the resources have labels/annotations to signal where they came from.

giving full control to the tenants means that they can add critical alerts without letting us know in advance, and i don't think that we are ready for that yet.